From 27f3586438835aa002ba9bb2dac6aba5fe225f17 Mon Sep 17 00:00:00 2001
From: Roman Inflianskas <rominf@aiven.io>
Date: Mon, 27 Jun 2022 11:27:08 +0300
Subject: [PATCH] pytest-localserver-0.6.0. rename aiosmtpd.Controller._stop

---
 pytest_localserver/smtp.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pytest_localserver/smtp.py b/pytest_localserver/smtp.py
index 3beb9f5..7e026c4 100644
--- a/pytest_localserver/smtp.py
+++ b/pytest_localserver/smtp.py
@@ -131,7 +131,7 @@ class Server(aiosmtpd.controller.Controller):
         #   be called more than once safely
         # - It passes the timeout argument to Thread.join()
         if self.loop.is_running():
-            self.loop.call_soon_threadsafe(self._stop)
+            self.loop.call_soon_threadsafe(self.cancel_tasks)
         if self._thread is not None:
             self._thread.join(timeout)
             self._thread = None
-- 
2.36.1

